### PR TITLE
mkcert: add page

### PR DESCRIPTION
**Pull request by Aracki**
I have added [mkcert](https://github.com/FiloSottile/mkcert).

### CODE REVIEW

File: pages/common/mkcert.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+- Generate certificate and private key for the given domain:
+
+`mkcert {{example.org}}`
+
+- Generate certificate and private key for the multiple domains:
+
+`mkcert {{example.org}} {{myapp.dev}} {{127.0.0.1}}`
+
+- Generate wildcard certificate and private key for the given domain:
```
Comment (agnivade) [inline pages/common/mkcert.md]: given domain _and its subdomains_ ?

Comment (Aracki) [reply pages/common/mkcert.md]: Resolved

File: pages/common/mkcert.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- Install the local CA in the system trust store:
+
+`mkcert -install`
+
+- Generate certificate and private key for the given domain:
+
+`mkcert {{example.org}}`
+
+- Generate certificate and private key for the multiple domains:
```
Comment (waldyrious) [inline pages/common/mkcert.md]:
```suggestion
- Generate certificate and private key for multiple domains:
```

Also, I'd suggest changing "the given domain" in the other examples to "a given domain", as that sounds more natural to me.

**Comment by waldyrious** [comment]
@mebeim looks like the suggested change "for the multiple domains" → "for multiple domains" was not applied. Please make sure that all comments are addressed before merging PRs. Not a big deal, of course, but it saves us an extra PR to fix the content.

**Comment by Aracki** [comment]
@waldyrious @mebeim Sorry, my bad. Should I just make another PR with this small change?

**Comment by waldyrious** [comment]
@Aracki no problem, we already did that in #2939. Just make sure to confirm before marking review comments as resolved in the future. 👍

**Pull request merged**